include/exadg/grid/grid_data.h

```diff
@@ -84,8 +98,8 @@ struct GridData
 
   unsigned int mapping_degree;
 
-  // TODO: path to a grid file
-  // std::string grid_file;
+  // path to a grid file
```

Comment:
This comment needs to be extended. E.g. 

"the filename needs to include a proper filename ending/extension so that we can internally deduce the correct type of file format."

Reply:
changed in d932484523cc4b0a6ccbe41963e79a51e5d74191.